tests/unit/amo/components/TestHomeHeroBanner.js

```diff
@@ -26,11 +35,63 @@ describe(__filename, () => {
     expect(root).toHaveClassName('HomeHeroBanner');
   });
 
+  it('renders with the "small" experiment classname', () => {
+    const root = shallowRender({ variant: AB_HOME_HERO_VARIANT_A });
+    expect(root).toHaveClassName('HomeHeroBanner--small');
+  });
+
+  it('renders without the "small" experiment classname', () => {
+    const root = shallowRender({ variant: AB_HOME_HERO_VARIANT_B });
+    expect(root).not.toHaveClassName('HomeHeroBanner--small');
+  });
+
   it('renders a carousel with random sections', () => {
     const root = shallowRender();
     const carousel = root.find(Hero);
 
     expect(carousel).toHaveLength(1);
     expect(carousel).toHaveProp('random', true);
   });
+
+  it('sends a tracking event when first rendered on the client', () => {
+    const fakeTracking = createFakeTracking();
+    const root = shallowRender({ _tracking: fakeTracking });
+
+    const { variant } = root.instance().props;
+
+    sinon.assert.calledWith(fakeTracking.sendEvent, {
+      action: variant,
+      category: `${AB_HOME_HERO_EXPERIMENT_CATEGORY} / Page View`,
+    });
+
+    sinon.assert.calledOnce(fakeTracking.sendEvent);
+  });
+
+  it('sends a tracking event on hero click', () => {
+    const fakeTracking = createFakeTracking();
+
+    const root = shallowRender({
+      _tracking: fakeTracking,
+    });
+
+    const { variant } = root.instance().props;
+
+    const heroBanner = root.find(Hero);
+
+    // We'll use the first item in the home heroes array as an example.
+    const heroItem = heroBanner.prop('sections')[0];
+    const firstHeroTitle = root.instance().getHeroes()[0].title;
+
+    const heroLink = shallow(heroItem).find('.HeroSection-link-wrapper');
+
+    expect(heroLink).toHaveLength(1);
+
+    heroLink.simulate('click', createFakeEvent());
+
+    sinon.assert.calledWith(fakeTracking.sendEvent, {
+      action: variant,
+      category: `${AB_HOME_HERO_EXPERIMENT_CATEGORY} / Click`,
+      label: firstHeroTitle,
+    });
+  });
```

Comment:
Can we add a `calledOnce`?

Reply:
I actually removed that b/c it's called once for the "page view" (on render) and once for the "click". So it would technically be calledTwice? I'll add that with a comment..?

Reply:
oh right, then nevermind

Reply:
Ok cool..I'll remove the calledTwice then